src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+:root {
+  font-weight: 300;
+}
```

Comment:
move it to html
use root here to variables

Reply:
Not fixed